pkg/alertmanager/types.go

```diff
@@ -119,7 +119,8 @@ type receiver struct {
 	MSTeamsV2Configs  []*msTeamsV2Config  `yaml:"msteamsv2_configs,omitempty"`
 	JiraConfigs       []*jiraConfig       `yaml:"jira_configs,omitempty"`
 	RocketChatConfigs []*rocketChatConfig `yaml:"rocketchat_configs,omitempty"`
-	MattermostConfigs []*mattermostConfig `yaml:"mattermost_configs,omitempty" json:"mattermost_configs,omitempty"`
+	MattermostConfigs []*mattermostConfig `yaml:"mattermost_configs,omitempty"`
+	IncidentioConfigs []*incidentioConfig `yaml:"incidentio_config,omitempty"`
```

Comment:
Hi @simonpasquier noticed when rebasing https://github.com/prometheus-operator/prometheus-operator/pull/8014 that was defined as `incidentio_config` instead of `incidentio_configs`. Fixed on the 8014 pr but just to note.